setup.py

```diff
@@ -49,6 +49,7 @@
         "gql==3.4.0",
         "graphql-core==3.2.3",
         "requests-toolbelt==0.10.1",
+        "snakemake==7.25.4",
```

Comment:
so I can install newer ones when they come out?
```suggestion
        "snakemake>=7.25.4",
```